setup.cfg

```diff
@@ -39,3 +39,96 @@ output_dir = opsdroid/locale
 [compile_catalog]
 domain = opsdroid
 directory = opsdroid/locale
+
+[metadata]
+name = opsdroid
+license = Apache License 2.0
+url = https://opsdroid.github.io/
+download_url = https://github.com/opsdroid/opsdroid/releases
+author = Jacob Tomlinson
+author_email = jacob@tom.linson.uk
+description = "An open source ChatOps bot framework."
+long_description = file: README.md
+long_description_content_type = text/markdown
+platforms = any
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Framework :: AsyncIO
+    Intended Audience :: Developers
+    Intended Audience :: System Administrators
+    Intended Audience :: Information Technology
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Communications :: Chat
+    Topic :: Scientific/Engineering :: Artificial Intelligence
+    Topic :: Software Development :: Libraries :: Python Modules
+keywords =
+    bot
+    bot-framework
+    opsdroid
+    botkit
+    python3
+    asyncio
+    chatops
+    devops
+    nlu
+
+[options]
+include_package_data = True
+zip_safe = False
+packages = find:
+test_suite = tests
+setup_requires = babel
+install_requires =
+	aiohttp==3.6.2
+	aioredis==1.3.1
+	aiosqlite==0.13.0
+	appdirs==1.4.4
+	arrow==0.15.6
+	Babel==2.8.0
+	bleach==3.1.5
+	certifi==2020.4.5.2
+	click==7.1.2
+	emoji==0.5.4
+	mattermostdriver==7.0.1
+	motor==2.1.0
+	multidict==4.7.6
+	nbconvert==5.6.1
+	nbformat==5.0.6
+	opsdroid-get-image-size==0.2.2
+	parse==1.15.0
```

Comment:
matcher-parse

Reply:
I would be ok with that, be interested to know what the others think.

Reply:
Maybe it would be best to do this with the `common` group @jacobtomlinson mentioned in chat, so it's possible to have a very very light opsdroid (maybe just using the webhook connector for instance) but we make all our default installation documentation be `pip install opsdroid[common]`.